spec/models/miq_server/update_management_spec.rb

```diff
@@ -186,7 +186,7 @@
       expect(LinuxAdmin::Rpm).to receive(:import_key).with("/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release").and_return(true)
       expect(LinuxAdmin::Rpm).to receive(:import_key).with("/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-beta").and_return(true)
       expect(EvmDatabase).to receive(:local?).and_return(true)
-      expect(yum).to receive(:update).once.with("cfme-appliance")
+      expect(File).to receive(:write).once.with(described_class::UPDATE_FILE, "cfme-appliance")
```

Comment:
this is ok but stubbing File has been a problem in the past since other code can also call methods on it that we're not setting expectations for... Can we let it write to this file and read it?  Then delete the file?